tests/unit/test_vmware_exporter.py

```diff
@@ -576,12 +577,20 @@ def test_vmware_get_inventory():
     datastore_2_folder.__dict__['childEntity'] = [datastore_2]
     datastore_2_folder.__dict__['name'] = 'datastore2-folder'
 
-    data_center_1 = mock.Mock()
+    data_center_1 = mock.Mock(spec=vim.Datacenter)
```

Comment:
Can we have a few more variants of this test with different folder structures? With different levels of nesting.

Reply:
Yes! Would be a good idea! I'd like to add improved testing at least for my cases in a different PR.